jdaviz/configs/default/plugins/virtual_observatory/vo_plugin.vue

```diff
@@ -0,0 +1,168 @@
+        label="Viewer"
+        :show_if_single_entry="true"
+        hint="Select a viewer to retrieve center coordinates, or Manual for manual coordinate entry."
+      />
+
+      <v-row v-if="viewer_selected !== 'Manual'">
+        <v-switch
+          v-model="coord_follow_viewer_pan"
+          label="Follow Viewer Center"
+          hint="Automatically adjust coordinates as viewer pans and zooms"
+          persistent-hint
+        ></v-switch>
+      </v-row>
+
+      <v-row>
+        <div :style="!(viewer_selected !== 'Manual' && !coord_follow_viewer_pan) ? 'width: 100%' : 'width: calc(100% - 32px)'">
+          <v-text-field
+            v-model="source"
+            label="Source/Coordinates"
+            hint="Enter a source name or coordinates in degrees to center your query on"
+            :disabled="viewer_selected !== 'Manual'"
+            :rules="[() => !!source || 'This field is required']"
+            persistent-hint>
+          </v-text-field>
+        </div>
+        <div v-if="viewer_selected !== 'Manual' && !coord_follow_viewer_pan" style="line-height:64px; width:32px">
+          <v-btn
+            id="autocenterbtn"
+            @click="center_on_data"
+            :disabled="viewer_centered"
+            icon>
+            <v-icon>
+              {{ viewer_centered ? 'mdi-crosshairs-gps' : 'mdi-crosshairs' }}
+            </v-icon>
+          </v-btn>
+        </div>
+      </v-row>
+
+      <v-row>
+        <v-select
+          v-model="coordframe_selected"
+          :menu-props="{ left: true }"
+          attach
+          :items="coordframes"
+          label="Coordinate Frame"
+          hint="Astronomical Coordinate Frame of the provided Coordinates"
+          :disabled="viewer_selected !== 'Manual'"
+          persistent-hint
+        ></v-select>
+      </v-row>
+
+    <j-plugin-section-header>Common Options</j-plugin-section-header>
+
+      <v-row justify="space-between">
+        <div :style="{ width: '55%' }">
+          <v-text-field
+            v-model.number="radius_val"
+            type="number"
+            label="Radius"
+            hint="Angular radius around source coordinates, within which to query for data (Default 1 degree)"
+            persistent-hint>
+          </v-text-field>
+        </div>
+        <div :style="{ width: '40%' }">
+          <v-select
+          v-model="radius_unit_selected"
+          attach
+          :items="radius_unit_items.map(i => i.label)"
+          label="Unit">
+          </v-select>
+        </div>
+      </v-row>
+
+
+    <j-plugin-section-header>Survey Collections</j-plugin-section-header>
+      <v-row>
+        <j-tooltip tipid='plugin-vo-filter-coverage'>
+        <span>
+          <v-btn icon @click.stop="resource_filter_coverage = !resource_filter_coverage">
+            <v-icon>mdi-filter{{ resource_filter_coverage ? '' : '-remove' }}</v-icon>
+          </v-btn>
+          Filter by Coverage
+        </span>
+        </j-tooltip>
```

Comment:
maybe this should just be a switch instead so that there is no ambiguity about whether its enabled or not.

Reply:
Done in https://github.com/spacetelescope/jdaviz/pull/2872/commits/0c5cdab4544b9e5e2ce0dce3299a87abe11aa27d